platformLibs/src/platform/osx/Hypervisor.def

```diff
@@ -0,0 +1,10 @@
+depends = darwin posix
+language = Objective-C
+package = platform.Hypervisor
+
+# Using headers instead of modules to workaround incorrect module definition in Xcode 12.2:
```

Comment:
Is there any kind of public issue on Apple bug tracker for it? If so, it's better to be referenced here.

Reply:
To the best of my knowledge, there is no public bug tracker for this.